about/using-github.md

```diff
@@ -33,13 +33,20 @@ If you're new to GitHub, you may want to read through the GitHub Help pages on [
 
 4. [Enable Two-factor authentication](https://github.com/settings/security)
 
-    - *This will soon be a requirement for membership in the @LLNL GitHub organization.*
-    - Two-factor authentication is set at your account level.
-    - You will be given the option to setup two-factor authentication with an app (such as Google Authenticator) or SMS.
-    - Selecting either option also provides you with a list of recovery codes to get into your account in the event you are locked out.
-    - If you setup with an app, you will still have the option to set a backup SMS number (recommended).
+    - *Membership in the @LLNL GitHub organization requires that 2FA has been enabled on your GitHub account*
```

Comment:
I still think we can't require this until employees have YubiKeys.  Authy looks ok but the instructions you link to below require snapshotting a QR code... also people have to install it.  Have you asked @davidbeckingsale what users are going to say to that?  I think it's too much of a barrier still.  Why not wait for YubiKeys?

Also how are you going to transition?  You can't grandfather people in -- turning on 2FA expels anyone without it.  You'll need to track people down and make them switch first...

Reply:
I think we should wait for YubiKeys.

Reply:
My complaint with waiting for YubiKeys is the estimated year long roll out that they still have ahead...

As for Authy, QR code is one option, but they also have an option to set up without that (It's "Enter Key Manually" in the screenshots on the page I linked to). To help solve the "downloading" problem (which doesn't seem like a big deal to me) I was thinking of trying to get it into the LanDesk and MacPatch catalogs to make it easier for our folks.

As for transitioning, yes, it would be tracking down all the users and getting them to switch, then enabling once everyone is converted. Same as we would have to do with a YubiKey only approach.

Reply:
Can we ask LLNL to prioritize GH org members?  Seems like a good way to promote out GitHub org and to get people on YubiKeys... also prevents users from having to switch twice.

Reply:
I can ask...

Reply:
In the meantime, there are a number of users that could start using Google Authenticator that would help cut the numbers down to something perhaps more reasonable to the YubiKey rollout folks: https://github.com/orgs/LLNL/people?utf8=%E2%9C%93&query=+two-factor%3Adisabled